tests/integration/components/settings/password-page-test.js

```diff
@@ -0,0 +1,10 @@
+import { test } from 'ember-qunit';
+import moduleForComponent from 'open-event-frontend/tests/helpers/component-helper';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('settings/password-section', 'Integration | Component | settings/password page');
```

Comment:
Change `Integration | Component | settings/password page ` to `Integration | Component | settings/password section` to match the component name